operator/k8s_identity.go

```diff
@@ -105,12 +116,27 @@ func identityGCIteration(ctx context.Context) {
 				}).Error("Deleting unused identity")
 				// If Context was canceled we should break
 				if ctx.Err() != nil {
+					if operatorOption.Config.EnableMetrics {
+						failedRuns++
+						metrics.IdentityGCRuns.WithLabelValues(metrics.LabelValueOutcomeFail).Set(float64(failedRuns))
+					}
```

Comment:
this should be done in line 130, i.e., after the for loop, otherwise if the operation fail, it will be counted as a `failedRun` and as a `successfulRun`.

Reply:
You're complete right. I have to put it  after loop. I just need to check `ctx.Err()` after the for loop to know if this GC run was `successful` or `failed`, right?